### ISSUE TITLE
access to api docs for directus 6.4.9

### ISSUE THREAD

**Issue by yud1z** [directus/directus#2236]
hi guys 

can i get api documentation for directus 6.4.9?

thank you

**Comment by vzool** [comment]
I completely agree @yud1z, not any one will jump from major version to another.
I suggest to provide multiple docs and branches dedicated to every major release with its docs and source code for that version.

I really like the organizing for what **Laravel** did with their every major releases:

![screenshot from 2018-11-06 10-32-48](https://user-images.githubusercontent.com/4952736/48049438-bf4fbb00-e1af-11e8-8256-0fc1bb122839.png)


And this is their docs:

![screenshot from 2018-11-06 10-34-27](https://user-images.githubusercontent.com/4952736/48049449-c8d92300-e1af-11e8-8e01-d6d97c2ca57e.png)

I hope this may clarify my point, Thanks :kissing_heart:

**Comment by shealavington** [comment]
Is this what you're looking for? https://github.com/directus/docs-6-legacy

**Comment by benhaynes** [comment]
The legacy v6 docs were in two repositories. The API Reference is here:

https://github.com/directus/api-docs-6-legacy

While I agree about the need for comprehensive docs moving forward, we simply can't support docs before version 7. We're a small, open source team and writing and maintaining docs for a system this complex is extremely time consuming. 

These legacy docs used a custom build script and are in a different format than the new VuePress docs for v7. We didn't have time to convert them. 

I like the idea of creating git releases for docs versions, we will plan on doing that for all minor/major releases, starting with v7.1. I don't know if VuePress supports this versioning, so previous versions will probably stay in git... but it would be easy to build them locally using a few commands.

Thanks for the recommendations!
